src/index.html

```diff
@@ -20,8 +20,372 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
```

Comment:
The href attribute for the navigation links is set to '#'. Consider providing meaningful URLs or using JavaScript to handle navigation if these are placeholders.